### DISCUSSION TITLE
What is Redis Commands on Billing page?

### DISCUSSION THREAD

**Discussion by yuntan** [vercel/community#3060] [Help] [answered]
### Summary

Can you help me understand the distinction between "Redis Commands" displayed on the Billing page and "KV Requests" on the Usage page? The Billing page shows 1,618,765 Redis Commands, while the Usage page reports 165,960 KV Requests. From the documentation, I gather there's an additional charge of 0.35 USD per 100,000 requests. I'm uncertain whether these "requests" refer to "Redis Commands" or "KV Requests." Can you clarify this for me?
https://vercel.com/docs/storage/vercel-kv/usage-and-pricing

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/574575/c46bec3b-b24a-46d4-95c1-ee750d7a8f06)

![image](https://github.com/vercel/community/assets/574575/d3cb3871-40a8-4ecd-9aeb-57142e89e577)

**Comment by amyegan** [comment]
Thanks for reporting this difference between usage numbers. The team is looking into it now 🙇

**Comment by yuntan** [comment]
Latest screenshots. 257k on Usage page and 48k on Billing page for same duration. Numbers on Billing page is reduced. What's happning?

![Usage](https://github.com/vercel/community/assets/574575/26698f5c-d380-41f0-ba74-549958bf758b)
![Billing](https://github.com/vercel/community/assets/574575/00b89584-5846-42f4-9e1e-fd3b4a6ceda3)

**Comment by ethomson** [comment, marked as answer]
Hi @yuntan - thanks for letting us know. It looks like there was a discrepancy updating the data in the dashboard. We believe that we've resolved this, can you take another look?

Thanks again!

**Reply by yuntan** [reply]
Currently on the Usage page, there are 358k KV Requests and on the Billing page, there are 208k Redis Commands. Since 150k requests are included in the Pro Plan, the remaining on the Billing page match the total number on the Usage page.

**Reply by ethomson** [reply]
Great - thanks for letting me know!

**Reply by ethomson** [reply]
Great - thanks for letting me know!

**Reply by ethomson** [reply]
Great - thanks for letting me know!

**Reply by ethomson** [reply]
Great - thanks for letting me know!

**Reply by ethomson** [reply]
Great - thanks for letting me know!
